als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/styler/astbuilder/DummyAstRawBuilder.scala

```diff
@@ -2,10 +2,10 @@ package org.mulesoft.als.suggestions.styler.astbuilder
 
 import org.mulesoft.als.common.YPartBranch
 import org.mulesoft.als.suggestions.{RawSuggestion, SuggestionStructure}
-import org.yaml.model._
 import org.mulesoft.common.client.lexical.{Position => AmfPosition}
+import org.yaml.model._
 
-class DummyAstRawBuilder(val raw: RawSuggestion)
+class DummyAstRawBuilder(raw: RawSuggestion)
```

Comment:
Use _override val raw: RawSuggestion_ would be more clear and readable for others

Reply:
Ok, it's done.